docs/installation/kubernetes.md

```diff
@@ -0,0 +1,40 @@
+## Kubernetes on Ubuntu
+
+There are multiple ways to run Kubernetes cluster with Ubuntu. Following lines explain
+how to deploy Kubernetes on Ubuntu on multiple public and private clouds, as well as
+bare metal.
+
+
+## Quick Start
+
+*conjure-up* provides a quick way to deploy Kubernetes on multiple clouds and 
+bare metal. It provides a user-friendlyUI that prompts you for cloud and configuration options:
+
+Available for Ubuntu 16.04 and newer:
+
+```sh
```

Comment:
@harshitagupta30
not able to get you?
do you mean removing this part and adding code?

i just followed what other .md files had.